src/objects/logo-group/logo-group.scss

```diff
@@ -0,0 +1,20 @@
+@use "../../compiled/tokens/scss/size";
+@use '../../mixins/ms';
+
+/**
+ * Because there is generous spacing between the logos, we added a lot of white
+ * space around Logo Group to help balance the pattern visually. Including the
+ * spacing as part of the pattern means we don't need to rely on utilities.
+ */
```

Comment:
This comment feels a little odd since there isn't any padding around the logo group. Maybe it should be rewritten and put closer to the items themselves (which have margin)?